scrapinghub/client.py

```diff
@@ -630,6 +632,8 @@ def update_tags(self, add=None, remove=None):
 
             >>> job.update_tags(add=['consumed'])
         """
+        if not (add or remove):
+            raise ValueError('Please provide tags to add or remove')
```

Comment:
Server responds with 200 for now, we may change it, but what's the point to do an empty request?